### DISCUSSION TITLE
Super Power League 4 size error

### DISCUSSION THREAD

**Discussion by superbonaci** [sanni/cartreader#845] [Hardware]
According to this video, when extracting the rom `SUPERPOWERLEAG4.sfc` it has size error according to `ROM Checker V1.53A`:
https://youtu.be/0YttRK9ZQW4?t=513

Is this a real issue? I can't test so I ask.

**Comment by breyell** [comment]
Your comment made me look into this and with some help from @Ancyker we found that the rom is being overdumped (3MB rather than 2MB). In the short-term if you use Advanced SNES ROM Utility to expand the rom to 4MB then split it into 2MB chunks, and rename the first file (with 0 in the name) from .bin to .sfc, that will be exactly what you're looking for.

**Reply by superbonaci** [reply]
Thanks for looking into it.
